rhpam-config/7.13.0/pim/process-migration.yaml

```diff
@@ -266,6 +266,20 @@ processMigration:
           kind: Service
           name: "[[.ApplicationName]]-process-migration"
         port:
-          targetPort: http
+          targetPort: "[[.ProcessMigration.TerminationRoute.Protocol]]"
+        #[[if .ProcessMigration.TerminationRoute.EnableTlsSection]]
+        tls:
+          insecureEdgeTerminationPolicy: Redirect
+          termination: "[[.ProcessMigration.TerminationRoute.Termination]]"
+          #[[if ne .ProcessMigration.TerminationRoute.Key ""]]
+          key: "[[.ProcessMigration.TerminationRoute.Key]]"
+          #[[end]]
+          #[[if ne .ProcessMigration.TerminationRoute.Certificate ""]]
+          certificate: "[[.ProcessMigration.TerminationRoute.Certificate]]"
+          #[[end]]
+          #[[if ne .ProcessMigration.TerminationRoute.CaCertificate ""]]
+          caCertificate: "[[.ProcessMigration.TerminationRoute.CaCertificate]]"
+          #[[end]]
+        #[[end]]
```

Comment:
indentation?
same for other similar additions.

Reply:
ok